FirebaseCore.podspec

```diff
@@ -29,6 +29,7 @@ Firebase Core includes FIRApp and FIROptions which provide central configuration
   s.private_header_files = 'Firebase/Core/Private/*.h'
   s.framework = 'Foundation'
   s.dependency 'GoogleUtilities/Logger', '~> 5.2'
+  s.dependency 'GoogleUtilities/Environment', '~> 5.2'
```

Comment:
@ryanwilson Not sure we catch missing dependencies in our tests...

Reply:
Nice catch - I'm not sure why this wouldn't be caught by `pod lib lint` - @paulb777  any ideas?

Reply:
Because it's always a transitive dep - https://github.com/firebase/firebase-ios-sdk/blob/master/GoogleUtilities.podspec#L38

Reply:
Makes sense! Can you please move it up one to be above `Logger` (alphabetical order)?